src/alerts_utils.cc

```diff
@@ -411,27 +392,39 @@ alert_save_state(zlistx_t *alerts, const char *path, const char *filename, bool
         return -1;
     }
 
-    // because serialization of fty_proto_zpl adds quotes, which have to removed around JSON,
-    // write into scratch file first
-    const char *suffix = "scratch";
     zconfig_t *state = zconfig_new ("root", NULL);
     fty_proto_t *cursor = (fty_proto_t *) zlistx_first (alerts);
 
     while (cursor) {
         fty_proto_print (cursor);
+
+        const char *alert_desc = fty_proto_description (cursor);
+        size_t alert_desc_size = strlen (alert_desc);
+        // new size is the next bigger or equal multiple of 4
+        size_t padded_size = (alert_desc_size + 3) & 0xFFFFFFFC;
+
+        uint8_t *padded_alert_desc = s_pad (alert_desc, padded_size);
+        size_t encoded_alert_desc_size = 1 + (5 * padded_size) / 4;
```

Comment:
Same question as above for `1+` and division rounding, asking just in case.

Reply:
Same answer :-)